index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+     <script src="https://kit.fontawesome.com/8c1870b6fa.js" crossorigin="anonymous"></script>
+    <title>Portfolio</title>
+  </head>
+  <body>
+    <!--header section start-->
+    <header>
+      <h2 class="logo"><a href="#">Fatima</a></h2>
+      <nav>
+        <div class="hamburger">
+          <svg width="750px" height="750px" viewBox="0 0 32 32" xmlns="http://www.w3.org/2000/svg" fill="none">
+            <path stroke="#6070FF" stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M5 8h22M5 16h22M5 24h22"/>
+          </svg>
+        </div>
+       
+      </nav>
+    </header>
+    <!--header section ends-->
+
+    <!--headline section starts-->
+    <section class="headline-main">
+      <section class="headline">
+        <h2 class="headline-heading">I'm Fatima.<br />Glad to see you!</h2>
+        <p class="headline-paragraph">
+          I'm a software developer! I can help you build a product , feature or
+          website Look through some of my work and experience! If you like what
+          you see and have a project you need coded, don't hestiate to contact
+          me.
+        </p>
+        <p class="headline-sm-suport">LET'S CONNECT</p>
```

Comment:
- [ ] Hi, nice work on your project. I noticed that you used a p tag for a subtitle on your web page. This is a subtitle for the icons at the bottom of it. I suggest that you use a header tag hx instead, where x is a number from 2 to 6 depending on the level of importance of your subtitle. The reason is that header tags are semantic elements that indicate the structure and hierarchy of your web page. They also help Google (and other search engines) understand the main topics and subtopics of your web page. Using header tags will also improve the accessibility and readability of your web page for users who use screen readers or have low vision. You can check this link for more info: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements. Please make this change and resubmit your project. Thank you for your cooperation. 😊